pkg/tekton/syntax/pipeline.go

```diff
@@ -1534,6 +1533,13 @@ func (j *ParsedPipeline) GenerateCRDs(pipelineIdentifier string, buildIdentifier
 			return nil, nil, nil, err
 		}
 
+		o := stage.Stage.Options
+		if o.RootOptions != nil {
+			if o.Retry > 0 {
+				stage.Stage.Options.Retry = s.Options.Retry
+				log.Infof("Generating CRDs with %d retries", stage.Stage.Options.Retry)
```

Comment:
Do we really want to be logging here? I tend to think we don't.

Reply:
Well, I think logging is good in general; is good for analysing possible bug or weird situations

Reply:
Then a better logging message makes sense - like, say "setting retries to %d for stage %s".